nikola/data/themes/base-jinja/templates/gallery.tmpl

```diff
@@ -15,12 +15,26 @@
     </p>
     {% endif %}
     {% if folders %}
-    <ul>
-    {% for folder, ftitle in folders %}
-        <li><a href="{{ folder }}"><i
-        class="icon-folder-open"></i>&nbsp;{{ ftitle|e }}</a></li>
-    {% endfor %}
-    </ul>
+        {% if galleries_use_thumbnail %}
+            {% for (folder, ftitle, fpost) in folders %}
+            <div class="thumnbnail-container">
+                <a href="{{ folder }}" class="thumbnail image-reference" title="{{ ftitle|e }}">\
```

Comment:
This backslash is invalid in Jinja2, and shouldn’t really be needed. I removed it.

Reply:
Thx!